src/AISmart.GAgent.Twitter/Agent/TwitterGAgentState.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using AISmart.Agent.GEvents;
+using AISmart.Agents;
+using Orleans;
+
+namespace AISmart.Agent;
+
+public class TwitterGAgentState : StateBase
+{
+    [Id(0)] public Guid Id { get; set; } = Guid.NewGuid();
+    [Id(1)] public string UserId { get; set; }
+    [Id(2)] public string Token { get; set; }
+    [Id(3)] public string TokenSecret { get; set; }
+    [Id(4)] public Dictionary<string, string> RepliedTweets { get; set; }
```

Comment:
If seems like you are caching a dict of replied tweets to process them when there is a response. Please clear up those entries that has already responded so that your state don't keep growing. If the user would like to revisit the list of replied tweets, he can do so through cqrs reading event logs